docs/en/sql-reference/statements/select/order-by.md

```diff
@@ -280,13 +280,15 @@ To fill multiple columns, add `WITH FILL` modifier with optional parameters afte
 
 ``` sql
 ORDER BY expr [WITH FILL] [FROM const_expr] [TO const_expr] [STEP const_numeric_expr], ... exprN [WITH FILL] [FROM expr] [TO expr] [STEP numeric_expr]
+[INTERPOLATE (col AS expr, ... colN AS exprN)]
```

Comment:
Let's not forget to add all implemented syntax: `INTERPOLATE`, `INTERPOLATE (col1, ... ,colN)`

Reply:
done